integrations/tensorflow/e2e/BUILD

```diff
@@ -17,173 +17,19 @@ load(
-[
-    iree_py_test(
-        name = "_".join([
-            "keras_vision_model",
-            data,
-            "top",
-            str(include_top),
-            model_name,
-            "test",
-        ]),
-        srcs = [
-            "keras_vision_model_test.py",
-        ],
-        args = [
-            "--model=%s" % model_name,
-            "--override_backends=%s" % backends,
-            "--data=%s" % data,
-            "--include_top=%d" % include_top,
-            "--url=https://storage.googleapis.com/iree_models/",
-        ],
-        main = "keras_vision_model_test.py",
-        python_version = "PY3",
-        tags = [
-            "external",
-            "large",
-            "manual",
-            "noga",
-            "notap",
-        ],
-        deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
-            "//integrations/tensorflow/bindings/python/pyiree/tf/support",
-        ],
-    )
-    # TODO(b/150244105) update backends with "tf,iree_vmla,iree_vulkan"
-    # and add other supported vision models
-    for data, include_top, model_name, backends in [
-        # "cifar10" has toy models with input 32x32, is good for debugging
-        # "imagenet" has real model weights for input 224x224
-        # TODO(rybakov) remove tests with include_top=0 as soon as include_top=1 works
-        ("cifar10", 1, "ResNet50", "tf"),
-        ("cifar10", 0, "MobileNet", "tf,iree_vmla"),
-        ("cifar10", 1, "MobileNet", "tf,iree_vmla"),
-        ("cifar10", 0, "MobileNetV2", "tf"),
-        ("cifar10", 1, "MobileNetV2", "tf"),
-        ("imagenet", 0, "ResNet50", "tf"),
-        ("imagenet", 1, "ResNet50", "tf"),
-        ("imagenet", 0, "MobileNet", "tf"),
-        ("imagenet", 1, "MobileNet", "tf"),
-        ("imagenet", 0, "MobileNetV2", "tf"),
-        ("imagenet", 1, "MobileNetV2", "tf"),
-    ]
-]
-
-[
-    iree_py_test(
-        name = name,
-        srcs = [name + ".py"],
-        python_version = "PY3",
-        # TODO(b/145815906) Get this running in OSS CI.
-        tags = ["noga"],
-        deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
-            "//integrations/tensorflow/bindings/python/pyiree/tf/support",
-        ],
-    )
-    for name in [
-        "conv_test",
-        "linspace_test",
-        "math_test",
-        # TODO(GH-1620): Re-enable this after fixing the failure on
-        # GitHub Actions.
-        "keras_lstm_static_test",
-    ]
-]
-
-# It is used to produce weights for keras vision models with input image size 32x32.
-# These models are not optimized for accuracy or latency (they are for debugging only).
-# They have the same neural net topology
-# with keras vision models trained on imagenet data sets
-py_binary(
-    name = "train_vision_models_on_cifar",
-    srcs = [
-        "train_vision_models_on_cifar.py",
-    ],
+iree_py_test_suite(
+    name = "e2e",
+    srcs = glob(["*.py"]),
+    excludes = ["linspace_test.py"],
```

Comment:
glob already takes an exclude argument. Also let's keep the glob constrained to the `_test.py` suffix as you do elsewhere

```suggestion
    srcs = glob(["*_test.py"], exclude = ["linspace_test.py"]),
```

Reply:
Done.